packages/client/hmi-client/src/model-representation/petrinet/petrinet-renderer.ts

```diff
@@ -219,7 +219,7 @@ export class PetrinetRenderer extends BasicRenderer<NodeData, EdgeData> {
 			.style('paint-order', 'stroke')
 			.style('fill', 'var(--text-color-primary')
 			.style('pointer-events', 'none')
-			.text((d) => d.id);
+			.text((d) => d.label);
```

Comment:
I would suggest that we pass-in a custom label functor, rather than assuming that everyone wants this change.

E.g. we could either pass it in the constructor, or have something like `renderer.setLabelFormatter` before the render call invocation.